src/main/java/net/minecraftforge/common/world/biomes/conditions/BiomeOrCondition.java

```diff
@@ -0,0 +1,39 @@
+package net.minecraftforge.common.world.biomes.conditions;
+
+import com.mojang.serialization.MapCodec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+import net.minecraft.world.biome.Biome;
+import net.minecraftforge.common.world.biomes.ForgeBiomeModifiers;
+import net.minecraftforge.common.world.biomes.conditions.base.BiomeConditionType;
+import net.minecraftforge.common.world.biomes.conditions.base.IBiomeCondition;
+
+public class BiomeOrCondition implements IBiomeCondition
+{
+    public static final MapCodec<BiomeOrCondition> CODEC = RecordCodecBuilder.mapCodec(inst ->
+            inst.group(
+                    IBiomeCondition.INNER_CODEC.fieldOf("first").forGetter(or -> or.first),
+                    IBiomeCondition.INNER_CODEC.fieldOf("second").forGetter(or -> or.second)
```

Comment:
Same here, using a list instead of two fields would reduce unnecessary nesting in json.